spec/menu_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'menu'
+
+describe Menu do
+
+  it 'should have a list of dishes with prices' do
+    expect(subject.view).to be_instance_of Hash
```

Comment:
Expectation clauses expecting a specific class is not such a good idea -> you might wanna change this later. Instead, construct the expected object and check us `eq`

```ruby
private

def test_menu_list
  {
    pizza: 2.0,
    pasta: 4.5,
    coke: 2.1
  }
end
```

And then in this test you can do

```ruby
it "lists dishes with prices"
  expect(subject.view_menu).to eq test_menu_list
end
```